### PR TITLE
ref(snuba): Add project_id to `issues` in request.

### PR DESCRIPTION
**Pull request by bretthoerner**
Part of SNS-117. Depends on https://github.com/getsentry/snuba/pull/97

### CODE REVIEW

File: src/sentry/utils/snuba.py [hunk 1]
```diff
@@ -350,8 +350,12 @@ def get_project_issues(project_ids, issue_ids=None):
     result = {}
     for h in hashes:
         tombstone_date = tombstones_by_project.get(h.project_id, {}).get(h.hash, None)
-        pair = (h.hash, tombstone_date.strftime("%Y-%m-%d %H:%M:%S") if tombstone_date else None)
-        result.setdefault(h.group_id, []).append(pair)
+        issue = (
+            h.hash,
+            h.project_id,
```
Comment (alex-hofsteede) [inline src/sentry/utils/snuba.py]: I think this is in the wrong place? IIUC the hashes should still be just 2-tuples of `(hash, tombstone)`? 

so the final collection would be something like `issues: [[issue_id, project_id, [[hash, tombstone], [hash2, tombstone2]]]]`

**Review by alex-hofsteede — CHANGES_REQUESTED** [review]

File: src/sentry/utils/snuba.py (hunk 1 above)
Comment (bretthoerner) [reply src/sentry/utils/snuba.py]: Haha, I totally flubbed that one.

Comment (alex-hofsteede) [reply src/sentry/utils/snuba.py]: no worries, that's what reviews are for

File: src/sentry/utils/snuba.py [hunk 2]
```diff
@@ -346,13 +346,16 @@ def get_project_issues(project_ids, issue_ids=None):
...
 
-    # return [(gid, [(hash, tombstone_date), (hash, tombstone_date), ...]), ...]
+    # return [(gid, pid, [(hash, tombstone_date), (hash, tombstone_date), ...]), ...]
     result = {}
     for h in hashes:
         tombstone_date = tombstones_by_project.get(h.project_id, {}).get(h.hash, None)
-        pair = (h.hash, tombstone_date.strftime("%Y-%m-%d %H:%M:%S") if tombstone_date else None)
-        result.setdefault(h.group_id, []).append(pair)
-    return list(result.items())[:MAX_ISSUES]
+        issue = (
```
Comment (alex-hofsteede) [inline src/sentry/utils/snuba.py]: minor naming nit, I still think this is maybe best called `pair` or something other than `issue` as it's not really an issue, but just a single hash/tombstone tuple.

**Review by alex-hofsteede — APPROVED** [review]
LGTM

**Pull request merged**